lib/src/main/scala/spinal/lib/eda/fusesoc/FuseSocGeneratorBuilder.scala

```diff
@@ -0,0 +1,68 @@
+package spinal.lib.eda.fusesoc
+
+import spinal.core._
+import com.fasterxml.jackson.core.{JsonFactory, JsonParser}
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.{YAMLFactory, YAMLGenerator}
+import com.fasterxml.jackson.module.scala._
+
+abstract class FuseSocGeneratorBuilder[P, C<:Component](componentName: String, defaultPram: P) {
```

Comment:
We normally use T prefix for a generic type. How about use TP and T instead of P and C?

Reply:
ok